### ISSUE TITLE
Add `--version` CLI option

### ISSUE THREAD

**Issue by ybiquitous** [arturbosch/detekt#2382]
Hello, I think more useful for many people if detekt would have a `--version` option to show the version to console.

## Expected Behavior

```shell-session
$ detekt --version
1.6.0
```

## Current Behavior

```shell-session
$ detekt --version
Was passed main parameter '--version' but no main parameter was defined in your arg class

Usage: detekt [options]
  Options:
    --auto-correct, -ac
...
```

## Context

For example, ktlint gets its version from its manifest file,

https://github.com/pinterest/ktlint/blob/9eb31ca6fbe1fd3f0fca708fafc15468218cecc5/ktlint/src/main/kotlin/com/pinterest/ktlint/internal/KtlintVersionProvider.kt#L16-L18

so I believe that detekt can show the version in a similar way.

What do you think about this suggestion?
Thank you for reading.

**Comment by ybiquitous** [comment]
Thank you for addressing this! I'm looking forward to the next release. 😊
